src/app/CriticalSectionImplPthread.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <cassert>
+#include <pthread.h>
+
+namespace chip {
+namespace Platform {
+
+#if CHIP_DEVICE_LAYER_TARGET_NRF5
+pthread_mutex_t eventLocker = Z_MUTEX_INITIALIZER;
```

Comment:
put these in an anonymous namespace (or mark static)

Reply:
Done, thanks